### ISSUE TITLE
Global Alerts Enhancement - presentation for handling multiple alerts

### ISSUE THREAD

**Issue by eyedefy** [vmware/clarity#295]
**Select one ...**  (check one with "x")
```
[ ] bug
[ ] feature request
[X ] enhancement
```

### Expected behavior
If the system has triggered two or more global alert messages, the Clarity global alert component gives me a way to know this to be the case. A hypothetical use-case might be: As a user, I need to know that my password is about to expire, I need to renew my user license, and I need to be aware that a dependent service will be shut down for maintenance during a specific window of time.

### Actual behavior
Currently it appears that Clarity only supports the presentation of one global alert message at any given point in time. 

### Suggested design/behavior

![globalalerts-multiple](https://cloud.githubusercontent.com/assets/945381/21533450/7c06cd7c-cd0f-11e6-84e4-98a614a43f1c.gif)
![serveralert3of3](https://cloud.githubusercontent.com/assets/945381/21533399/c61161f8-cd0e-11e6-96f5-1400998eae61.png)
![password-alert2of3](https://cloud.githubusercontent.com/assets/945381/21533400/c61200b8-cd0e-11e6-9e47-44228d6dd93a.png)
![licence-alert1of3](https://cloud.githubusercontent.com/assets/945381/21533401/c61498d2-cd0e-11e6-83b5-edcdef735369.png)

**Comment by yenmade** [comment]
Thanks for this awesome concept for handling multiple global alerts @eyedefy! We will queue this on the UX backlog to enhance global alerts.

**Comment by youdz** [comment]
## API of this component

This should be a simple component wrapping alerts: 
```html
<clr-alerts>
    <clr-alert *ngFor="let alert of alerts">...</clr-alert>
</clr-alerts>
```
or 
```html
<clr-alerts>
    <clr-alert>...</clr-alert>
    <clr-alert>...</clr-alert>
    <clr-alert>...</clr-alert>
</clr-alerts>
```
(Make sure to test the `*ngFor` case.)

It should offer two-way binding on the current `Alert` component being displayed, as well as the current index of the alert being displayed:
```html
<clr-alerts [(clrCurrent)]="myCurrentAlert">
```
and 
```html
<clr-alerts [(clrCurrentIndex)]="myCurrentAlertIndex">
```
Do not worry about support them both simultaneously, they would conflict. We're offering both, but consumers should use one or the other depending on their use case.

For now, the `< 1 of 3 >` pagination will not be customizable. You can write a specific component for it that will be used internally, and we can consider making it flexible and public in the future, but it's not part of the MVP. **As such, the "of" word cannot be part of the default pagination, because it would not be i18n'd. For now, the pagination should display `< 1/3 >`, or anything else that would convey the same message without text.** Please double-check with @eyedefy about this compromise.

**Comment by victormejia** [comment]
@yenmade @youdz I might be looking into this one, would the pagination component be a specific width/percentage?

**Comment by youdz** [comment]
@victormejia: Slight screw-up on our part, we forgot to assign the external contributor who picked it up last week. So please don't start working on this, or one of you will have written it for nothing. 😅

**Comment by youdz** [comment]
(Oh right, there is no private messaging on GitHub)

@victormejia: I understand you're going through our `contributions-welcome` issues, but we don't have many of these yet. I could do a quick pass tomorrow to find a few more, clean them up and post a precise spec on them. Would that work for you? 
Ideally, after that quick pass I'll go more thoroughly through them and document every single one that could be contributed to, but that's going to take much longer than picking the low hanging ones.

**Comment by victormejia** [comment]
@youdz sounds good! I didn't start working on this, was just looking at what I can tackle next. Looking forward to working on new issues 😃

**Comment by youdz** [comment]
Currently blocked by https://github.com/vmware/clarity/issues/692.

Since we have no idea how long this bug will take to be fixed, we might have to come up with a workaround.
